### DISCUSSION TITLE
No update for bagisto version received.

### DISCUSSION THREAD

**Discussion by soodankush** [bagisto/bagisto#6041] [Q&A]
# Bug Report

## Issue Description

Currently, I have not received any notification regarding the version update.  My project is on version 1.20 and I am not able to update it new version. 

## Bagisto Version

- [ ] master branch
- [ ] 1.3.3
- [ ] 1.3.2
- [ ] 1.3.1
- [ ] 1.3.0
- [ -] 1.2.0

## Actual Result

Getting errors in admin panel w.r.t. to  updating the project from 1.2.0 to 1.3.*
## Expected Result

I should be able to update my store

**Comment by Sanjaybhattwebkul** [comment]
Hi @soodankush 
Can you please elaborate on what issues are you getting exactly

**Comment by soodankush** [comment]
Hi @Sanjaybhattwebkul ,

I am not sure whether should I update from ver 1.2.0 because I have tried in my local and getting way too many errors.

What do you suggest?? Should I do it or not ???
Which is the stable version which I should update to from 1.2.0

**Comment by Sanjaybhattwebkul** [comment]
Hi @soodankush 
Have you done any **core customization**?

**Comment by soodankush** [comment]
@Sanjaybhattwebkul,
I have only added the Razorpay package for payment. Rest there is no customization.
What do u say? Should I proceed

**Comment by Sanjaybhattwebkul** [comment]
@soodankush 
**Yes** you can process, **but** the **First Razorpay package** has to **be compatible** for the V1.3.3

**Comment by soodankush** [comment]
Okay, @Sanjaybhattwebkul.
Thanks. I will try and keep you updated.
